src/Extension/CwpControllerExtension.php

```diff
@@ -135,10 +140,10 @@ protected function triggerBasicAuthProtection()
         });
 
         if ($existingMemberCanAccessUAT) {
-            $allowed = array(
```

Comment:
While I agree with these sorts of changes...I do always discourage colleagues from making unrelated changes becuase it makes reviews harder, to sort the wheat from the chaff whne diffinfg

Reply:
You're totally right!